public/nativedisplay.h

```diff
@@ -244,6 +244,12 @@ class NativeDisplay {
     return false;
   }
 
+  /**
+  * API for setting canvas color for display.
+  */
```

Comment:
we usually add the name of the variable and comment it out.

Can you explain the expectations of the inputs to this function

Reply:
Ok, will include this feedback in v2.

Reply:
hmm we should just take in r,g,b,a values here. Converting that to something kernel driver understands should be internal logic in hwc side. btwn We could move the drmhelper functions to within hwc.

Reply:
Yeah, now that I think about it, your suggestion seems like a better approach. Will send out a new version...